template/__tests__/App.test.tsx

```diff
@@ -3,11 +3,9 @@
  */
 
 import React from 'react';
-import ReactTestRenderer from 'react-test-renderer';
+import { render } from "@testing-library/react-native";
 import App from '../App';
 
 test('renders correctly', async () => {
-  await ReactTestRenderer.act(() => {
-    ReactTestRenderer.create(<App />);
-  });
+  render(<App />);
```

Comment:
```suggestion
  render(<App />);
  expect(screen.getByText('Read the docs to discover what to do next')).toBeOnTheScreen();
```

Reply:
All Jest test should contain some `expect` assertions. A typical one is that a certain element is rendered on the screen. I've picked a sample text from `App.tsx` here.